### DISCUSSION TITLE
Coverage measured against changed files.

### DISCUSSION THREAD

**Discussion by OssamaSijbesma** [vitest-dev/vitest#4881] [Ideas]
Related to #4868 and #4057

**Current situation:**
Currently the [changed](https://vitest.dev/guide/cli.html#changed) option only works in combination with running tests and not when measuring coverage. However, it is possible to run the coverage measurements alongside the changed option causing it to display faulty outcomes.

**Preferable situation:**
The coverage reports is limited to the changed files, similar to using the [vitest related](https://vitest.dev/guide/cli.html#vitest-related) command in combination with the coverage option.

**Target audience:**
Users with a lot of tests.

**Comment by jackonwei** [comment]
Hey, I have a question, I want to know how you do it, after the coverage report is exported, only a single test of one file is written, but the other files are 100% by default, and the overall project project coverage does not include the untested files, only the coverage of the written single test file is calculated, which is not what I want
